public/UniApplyBot/index.html

```diff
@@ -41,9 +43,10 @@
             <h1>Hi, it's <span>UniApplyBot</span></h1>
             <h3 class="text-animation"> I'm here to help with <span></span></h3>
             <p>
-                Lorem ipsum dolor sit amet consectetur adipisicing elit. 
-                Perspiciatis quaerat ratione animi non iure nesciunt dicta repellat. 
-                Fugiat modi tempora quidem consectetur, libero doloremque adipisci aspernatur eius quasi qui veniam.
+                UniApplyBot is your personal assistant designed to streamline university applications. Whether you're
+                searching for the right course, seeking advice on requirements, or tracking your application progress,
+                UniApplyBot is here to provide you with real-time support and guidance.
+
```

Comment:
"Tracking application progress, advice on requirements", these two features are features that are not catered in the chatbot, unless they must be incorporated. I believe today we win with the multilingual feature so we can progress clearly.